draft-bbs-signatures.md

```diff
@@ -382,27 +380,25 @@ This operation checks if a public key is valid.
 As an optimization, implementations MAY cache the result of KeyValidate in order to avoid unnecessarily repeating validation for known public keys.
 
 ```
-result = KeyValidate(PK)
+result = KeyValidate(W)
 
 Inputs:
 
-- PK (REQUIRED), an octet string of the form outputted by the SkToPk operation.
+- W (REQUIRED), a public key as calculated at step one (1) in SkToPk.
 
 Outputs:
 
 - result, either VALID or INVALID.
 
 Procedure:
 
-1. W = octets_to_point_g2(PK)
-
-2. if W is INVALID, return INVALID
+1. if W is INVALID, return INVALID
```

Comment:
As per above I think we should consider drawing the boundary for this operation as `W` being the valid output from the SkToPk operation

Reply:
I'm not opposed to that. Just to clarify though, do you mean SkToPk not returning octetcs but W directly??